.gitmodules

```diff
@@ -5,3 +5,4 @@
 [submodule "test/tests"]
 	path = test/jsontests
 	url = https://github.com/ethereum/tests.git
+  branch = invalid-ops
```

Comment:
I can remove it.  But until the invalidOps tests are merged into tests/master, I have to point at the right tests branch.